src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/AuthService.kt

```diff
@@ -9,7 +9,7 @@ import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.WebClientResponseException
 import org.springframework.web.reactive.function.client.awaitBody
-import uk.gov.justice.digital.hmpps.manageusersapi.resource.CreateRole
+import uk.gov.justice.digital.hmpps.manageusersapi.resource.RoleDescriptionAmendment
```

Comment:
This should no longer be in here if the merge went ok. RoleDescriptionAmendment is no longer in auth and is on main

Reply:
Now removed